recipes/sos-rmarkdown/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sos-rmarkdown" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d67523c57e41b522bef91078565b616bfceeec66b7d250acd965e73d479f743b
+
+build:
+  number: 0
+  skip: True  # [py2k]
```

Comment:
Please remove this line, and add `noarch: python` instead.

Reply:
This package cannot be no-arch because it uses `sos`, which only supports python 3.